README.md

```diff
@@ -1,25 +1,75 @@
-# must-gather for Red Hat OpenShift Data Science
+# must-gather for Red Hat OpenShift AI
 
 The must-gather script allows a cluster admin to collect information about various key resources and namespaces
-for Red Hat OpenShift Data Science.
+for Red Hat OpenShift AI.
 
 ## Data Collected
 
-The must-gather script currently collects data from following namespaces
+The must-gather script currently collects data from following default namespaces
 
 - redhat-ods-operator
 - rhods-notebooks
 - redhat-ods-applications
 - redhat-ods-monitoring
 
-The script collects data from all the namespaces that has -
-- `DataSciencePipelinesApplication` instances
+and datasciencecluster and dscinitialization instances from cluster
```

Comment:
Are `Event` objects collected too? For all default namespaces?

Reply:
no. i can add Event as default

Reply:
ah, ok, i can see Event is auto created by must-gather, in these 4 namespaces and "default" namespace